rest/context_test.go

```diff
@@ -97,3 +132,35 @@ func TestHeader(t *testing.T) {
 
 	assert.Equal(req.Header, ctx.Header())
 }
+
+func TestBuildURL(t *testing.T) {
+	assert := assert.New(t)
+
+	api := NewAPI(NewConfiguration())
+	api.RegisterResourceHandler(TestResourceHandler{})
+	api.RegisterResourceHandler(ComplexTestResourceHandler{})
+
+	req, _ := http.NewRequest("GET", "https://example.com/api/v1/widgets", nil)
+	gContext.Set(req, "version", "1")
```

Comment:
I don't see any versions other that `v1`. Would it make sense to add coverage over other versions like `v2` in this test?